shared/teams/really-leave-team/last-owner.tsx

```diff
@@ -31,15 +32,20 @@ const ReallyLeaveTeam = (props: Props) => (
         You can't leave the {props.name} team because you're the only owner.
       </Kb.Text>
       <Kb.Text type="Body" center={true} style={styles.bodyText}>
-        You'll have to add another user as an owner before you can leave {props.name}.
+        You'll have to add another user as an owner before you can leave {props.name}, or{' '}
+        <Kb.Text type="BodyPrimaryLink" onClick={props.onDeleteTeam}>
+          delete the&nbsp;team
```

Comment:
maybe just "delete the team" here

Reply:
Do you mean drop the non-breaking space?

Reply:
yeah, not sure why it's needed?

Reply:
Oh, it's to avoid a widow on mobile.

Reply:
gotcha, sounds good then!